src/single-spa-react.js

```diff
@@ -283,9 +283,16 @@ function getElementToRender(opts, props, mountFinished) {
       )
     : rootComponentElement;
 
-  if (opts.errorBoundary) {
+  if (
+    opts.errorBoundary ||
+    props.errorBoundary ||
```

Comment:
Good shout putting `opts` first here, presumably that means that an errorBoundary option passed to `singleSpaReact({...` would trump any prop passed from `singleSpa.registerApplication({ customProps: ...`, which makes sense to me 👍

Reply:
Yes that's what it would mean - I'm thinking that the opts should take priority in case someone wants to override what comes from the root config.